frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -935,6 +935,24 @@ const cardList = handleActions(
   null,
 );
 
+// can do more here to
+function newDashboard(before, after) {
+  const dashboard = { ...before, ...after, isDirty: true };
+  const newParamSlugs = after.parameters && _.pluck(after.parameters, "slug");
+  if (
+    newParamSlugs &&
+    !_.every(_.keys(before.embedding_params), embedId => {
+      return _.contains(newParamSlugs, embedId);
+    })
+  ) {
+    dashboard.editWarning = dashboard.editWarning =
```

Comment:
I think ideally this `editWarning` wouldn't be part of the dashboard itself (it will get sent to the server when saving, I think). Maybe a separate reducer.